### PR TITLE
ADD: unit

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: invoice/models.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def items(self):
     @property
     def table_export(self):
         """Get the line items as list and export all of them as table with a header row."""
-        header = ['Name', 'Description', 'Quantity', 'Price', 'Tax', 'Net Total', 'Total']
+        header = ['Name', 'Description', 'Quantity', 'Unit', 'Price', 'Tax', 'Net Total', 'Total']
```
Comment (iTitus) [inline invoice/models.py:154]: Can we combine unto with quantity, like the money amounts?
Would save some space in the table.

**Review by iTitus — CHANGES_REQUESTED** [review]

File: invoice/models.py (hunk 1 above)
Comment (Segelzwerg) [reply invoice/models.py:154]: oh yeah, great idea

File: invoice/models.py [hunk 2]
```diff
@@ -241,8 +242,9 @@ def price_string(self) -> str:
 
     @property
     def quantity_string(self) -> str:
-        """Get the quantity string."""
-        return f'{self.quantity:.4f}'.rstrip('0').rstrip('.,')
+        """Get the quantity string including the unit."""
+        quantity = f'{self.quantity:.4f}'.rstrip('0').rstrip('.,')
+        return f'{quantity} {self.unit}'
```
Comment (iTitus) [inline invoice/models.py:247]:
```suggestion
        quantity = f'{self.quantity:.4f}'.rstrip('0').rstrip('.,')
        if self.unit:
            return f'{quantity} {self.unit}'
        return quantity
```

File: invoice/templates/invoice/invoice_form.html [hunk 3]
```diff
@@ -70,6 +71,7 @@
                                 <td>{{ item.name }}</td>
                                 <td>{{ item.description }}</td>
                                 <td class="text-end">{{ item.quantity_string }}</td>
+                                <td class="text-end">{{ item.unit }}</td>
```
Comment (iTitus) [inline invoice/templates/invoice/invoice_form.html:74]:
```suggestion
```

File: invoice/templates/invoice/invoice_form.html [hunk 4]
```diff
@@ -58,6 +58,7 @@
                             <th scope="col">Name</th>
                             <th scope="col">Description</th>
                             <th scope="col">Quantity</th>
+                            <th scope="col">Unit</th>
```
Comment (iTitus) [inline invoice/templates/invoice/invoice_form.html:61]:
```suggestion
```

**Review by iTitus — CHANGES_REQUESTED** [review]

**Review by iTitus — APPROVED** [review]

**Pull request merged**